### DISCUSSION TITLE
WindowTop causes significant background CPU usage while idling

### DISCUSSION THREAD

**Discussion by WindowsTablet** [WindowTop/WindowTop-App#368] [General]
WindowTop is a great utility, but it has one major problem. It idles in the background with significant CPU usage, even when none of its features are running, eating my battery away. It consistently shows at least 3% background in Windows "Battery usage" settings each day.
 
Could anything be done to improve this?

Thanks in advance.

**Comment by gileli121** [comment]
See my comment on
https://github.com/WindowTop/WindowTop-App/issues/369

We will discuss it there.
